DocumentationGuide.md

```diff
@@ -0,0 +1,183 @@
+- [Serialized Fields](#serialized-fields)
+- [Enumeration Values](#enumeration-values)
+
+In addition to the above, the code should be well commented to allow for maintenance, bug fixes and ease of customization. 
+
+### Class, Struct, Enum Summary Blocks
+If a class, struct or enum is being added to the MRTK, it's purpose must be described. This is to take the form of a summary block above the class.
+
+```
+    /// <summary>
+    /// AudioOccluder implements IAudioInfluencer to provide an occlusion effect.
+    /// </summary><< >>
+```
+
+If there are any class level dependencies, they should be documented in a remarks block, immediately below the summary.
+
+```
+    /// <remarks>
+    /// Ensure that all sound emitting objects have an attached AudioInfluencerController. 
+    /// Failing to do so will result in the desired effect not being applied to the sound.
+    /// </remarks>
+```
+
+Pull Requests submitted without summaries for classes, strutures or enums will not be approved.
+
+### Property, Method, Event Summary Blocks
+As with classes, structures and enums, PMEs (and fields) are to be documented with summary blocks, regardless of code visibility (public, private, protected and internal). The documentation generation tool is responsible for filtering out and publishing only the public and protected features.
+
+NOTE: A summary block is **not** required for Unity methods (ex: Awake, Start, Update).
+
+PME documentation is **required** for a Pull Request to be approved.
+
+As part of a PME summary block, the meaning and purpose of parameters and returned data is required.
+
+```
+        /// <summary>
+        /// Sets the cached native cutoff frequency of the attached low pass filter.
+        /// </summary>
+        /// <param name="frequency">The new low pass filter cutoff frequency.</param>
+        /// <returns>The new cutoff frequency value.</remarks>
+
+```
+
+[Go Up](#source-documentation)
+
+### Feature Introduction Version and Dependencies
+As part of the API summary documentation, information regarding the MRTK version in which the feature was introduced and any dependencies should be documented in a remarks block.
+
+Dependencies should include extension and/or platform dependencies.
+
+```
+    /// <remarks>
+    /// Introduced in MRTK version: 2018.06.0 
+    /// Requires installation of: ImaginarySDK v2.1
+    /// Minimum Operating System: Windows 10.0.11111.0
+    /// </remarks>
+```
+
+[Go Up](#source-documentation)
+
+### Serialized Fields
+It is a good practice to use Unity's Tooltip attribute to provide runtime documentation for a script's fields in the Inspector.
+
+So that configuration options are included in the API documentation, scripts are required to include *at least* the tooltip contents in a summary block.
+
+```
+        /// <summary>
+        /// The quality level of the simulated audio source (ex: AM radio).
+        /// </summary>
+        [Tooltip("The quality level of the simulated audio source.")]
```

Comment:
I wish there was an easy way to just add the tooltip as the documentation.

Reply:
Yeah. Me too.